step-capstone/src/App.js

```diff
@@ -1,15 +1,60 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from "react-router-dom";
 import './styles/App.css';
-import TravelObject from './components/TravelObject'
 
 class App extends React.Component {
+  isLoggedIn = true; 
+  handleLogin(){
+    if(this.isLoggedIn){
+      return(
+        <Redirect to = "/trip-list"/>
+      ); 
+    } else {
+      return (
+        <Redirect to = "/login-page"/>
+      );
+    }
+  }
   render () {
     return(
       <div className="App">
-        <TravelObject type="flight" />
+        <Router>
+          {this.handleLogin()}
+          <Switch>
+            <Route path="/trip-list">
+              <TripList userId = 'userId' />
+            </Route>
+            <Route path="/login-page">
+              <Login />
+            </Route>
+          </Switch>
+        </Router>
+      
       </div>
     );
   }
 }
+function Login(){
+  return ( 
+    <Login/>
```

Comment:
Would rendering itself cause some error?

Reply:
probably that was just a placeholder till I get Dan's authentication stuff. happened to use the same name